wiki/Ranking_Criteria/osu!mania/en.md

```diff
@@ -62,9 +62,9 @@ Overall rules and guidelines apply to every kind of osu!mania difficulty. Rhythm
 - **If multiple key modes are present in a single beatmap, the key mode must be denoted in all difficulty names. Otherwise, the key mode must not be denoted.**
 - **Beatmaps using the special N+1 style of mapping must be denoted as such under the `Advanced` tab of song setup in the editor and "N+1" must be added to the tags. For beatmaps containing multiple key modes, difficulties using this style must be denoted using N+1 notation (e.g. using 7K+1 instead of 8K on difficulty names).**
 - **If the [drain time](/wiki/Beatmap/Drain_time) of each difficulty is...**
-  - **...lower than 2:30**, the lowest difficulty of each included keymode cannot be harder than a Normal, **OR** each keymode must provide a spread starting at least 3 difficulty levels below the highest difficulty.
-  - **...between 2:30 and 3:15**, the lowest difficulty of each included keymode cannot be harder than a Hard, **OR** each keymode must provide a spread starting at least 2 difficulty levels below the highest difficulty.
-  - **...between 3:15 and 4:00**, the lowest difficulty of each included keymode cannot be harder than an Insane, **OR** each keymode must provide a spread starting at least 1 difficulty level below the highest difficulty.
+  - **...lower than 2:30**, the lowest difficulty of each included keymode cannot be harder than a Normal, OR each keymode must provide a proper spread containing at least 4 difficulties (including the highest difficulty).
+  - **...between 2:30 and 3:15**, the lowest difficulty of each included keymode cannot be harder than a Hard, OR each keymode must provide a proper spread containing at least 3 difficulties (including the highest difficulty).
+  - **...between 3:15 and 4:00**, the lowest difficulty of each included keymode cannot be harder than an Insane, OR each keymode must provide a proper spread containing at least 2 difficulties (including the highest difficulty).
```

Comment:
what exactly is a "proper spread"? if we are using vague terms like this, they should be properly defined, however a footnote for this seems unnecessary, so is there a better phrasing of what you mean? perhaps could do:

```suggestion
  - **...lower than 2:30**, the lowest difficulty of each included keymode cannot be harder than a Normal, OR each keymode must provide a spread containing at least 4 difficulties (including the highest difficulty) without skipping any difficulty levels.
  - **...between 2:30 and 3:15**, the lowest difficulty of each included keymode cannot be harder than a Hard, OR each keymode must provide a spread containing at least 3 difficulties (including the highest difficulty) without skipping any difficulty levels.
  - **...between 3:15 and 4:00**, the lowest difficulty of each included keymode cannot be harder than an Insane, OR each keymode must provide a spread containing at least 2 difficulties (including the highest difficulty) without skipping any difficulty levels.
```

Reply:
The point of the clarification is exactly to avoid difficulty levels (given a level or a name are different terms in mania). A spread is defined equally between all modes but mania adds the specific distinction `On difficulties Insane and harder, a proper spread is defined by relative difficulty gaps similar to those formally defined between lower difficulties`

I think linking that into the `spread` word would be a nice addition though.

Reply:
Although now that I see it closely, its literally the next sentence so probably not needed.

Reply:
The point of this PR was to get rid of the term "difficulty level" since that implies it must fall within ENHIX conventions. Currently, mania is allowed to have X only spreads under this rule, and by reintroducing "difficulty level" into the language, we come back to the confusion that technically, an X diff would need at minimum, an I diff to be rankable when that's not been the common interpretation within mania.